source/documentation/deploying_services/index.md

```diff
@@ -4,26 +4,37 @@
 
 Many 12-factor applications rely on backing services such as a database, an email delivery service or a monitoring system. Routing services can be used to proxy and perform preprocessing on application requests such as caching, rate limiting or authentication.
 
-In Cloud Foundry, backing and routing services are referred to as 'services' and are available through the Cloud Foundry ``cf marketplace`` command.
+In Cloud Foundry, backing and routing services are referred to as 'services' and are available through the Cloud Foundry ``cf marketplace`` command. GOV.UK PaaS enables you to create a database service and bind it to your app. 
 
 Currently, the following database services are available in the marketplace:
 
  - PostgreSQL
  - MySQL
  - MongoDB (currently only available on private beta)
 
-
-Each service in the marketplace can have multiple plans available. For example, there are different PostgreSQL plans which vary by availability, storage capacity and encryption.
+Each service in the marketplace can have multiple plans available with different characteristics. For example, there are different PostgreSQL plans which vary by availability, storage capacity and encryption.
 
 Users can also define their own external services that are not available in the marketplace by using [User-Provided Service Instances](/#user-provided-service-instance).
 
+## PostgreSQL
+
+PostgreSQL is an object-relational database management system. It is open-source and designed to be extensible; currently the postgis and uuid-ossp extensions are enabled.
+
+## MySQL
+
+MySQL is an open source relational database management system that uses Structured Query Language (SQL) and is backed by Oracle.
+
+## MongoDB
```

Comment:
This position of these H2 headings makes `### Paid service plans` a child of `## MongoDB` instead of `## Services and plans` which it was previously, which I think is incorrect.

I don't have a good suggestion of how to solve that, because making them H3s would mean that they don't appear in the left-hand navigation, which @tomskerous was really keen on doing.

Any thoughts?